.github/pull_request_assignment.yml

```diff
@@ -103,108 +95,24 @@
     paths:
       - "specification/network/**"
     reviewers:
```

Comment:
this seems to be for both data and control plane?

Reply:
The bot will take the first matching rule so the first one will catch all the mgmt plane specs. 

Given network only has resource-manager, https://github.com/Azure/azure-rest-api-specs/tree/main/specification/network I suspect this rule can simply be removed.

@raych1 any idea why the branches setting is on this rule?

Reply:
ok lets remove if redundant

Reply:
It's not redundant, customized branch for PR assignment is only supported by this approach currently. If all below rules are removed, then the PR targeting this customized branch would not be assigned.